event_listener/dfusion_db/snapp_event_receiver.py

```diff
@@ -64,9 +61,9 @@ def __update_accounts(self, transition: StateTransition) -> None:
                     balances[index] -= datum.amount
                 else:
                     self.logger.info("Insufficient balance: account {} - token {} for amount {}".format(
-                        datum.account_id, datum.token_id, datum.amount))
-            elif transition.transition_type == TransitionType.Auction:
-                pass
+                        datum.account_id,
+                        datum.token_id,
+                        datum.amount))
```

Comment:
For Line breaking of this type, I would propose the following
```python
self.logger.info(
    "Insufficient balance: account {} - token {} for amount {}".format(
        datum.account_id, 
        datum.token_id, 
        datum.amount
    )
)
```

Reply:
Can you suggest in the PR so I only have to accept?

Reply:
yes sure!